eureka2-core/src/main/java/com/netflix/eureka2/utils/rx/PauseableSubject.java

```diff
@@ -0,0 +1,169 @@
+package com.netflix.eureka2.utils.rx;
```

Comment:
It is a copy of NotificationsSubject, but not tied to ChangeNotification type. No changes are done to it other then type parameters. I will remove NotificationsSubject in a separate commit, as it will touch a lot of places in the code.